sos/plugins/manila.py

```diff
@@ -14,31 +14,28 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
+import os
 from sos.plugins import Plugin, RedHatPlugin
 
 
 class ManilaPlugin(Plugin, RedHatPlugin):
     """Openstack Manila"""
     plugin_name = "manila"
     profiles = ('openstack', 'openstack_controller', 'openstack_manila')
-    option_list = [("cmds", "gathers openstack manila commands", "slow", False)]
+    option_list = [("nas", "List NAS shares", "fast", False)]
```

Comment:
"manila list" command requires user/tenant environment variable need to be set like for other openstack plugins.